first-milestone

tta2yta opened the pull request:
we created a branch called "feature" for the development of the application.
On the initial commit, we created a basic structure for the page and added bootstrap links, CDN and Font awesome, as well as set up linter configured it.

In this section, we have done the Header and the footer section of thenextweek website.
The "Mobile first" approach is used in order to achieve responsiveness principles.
Elements are styled using bootstrap classes and custom CSS file
The issues raised from the first review of the project in the dashboard are fully addressed.

Please review my project before I merge my branch.
Thanks in advance!

tta2yta committed:
initial commit

tta2yta committed:
adding bootstrap

tta2yta committed:
adding header large screen

tta2yta committed:
adding header small screen

tta2yta committed:
adding styles header

tta2yta committed:
adding media queries

tta2yta committed:
adding top menu

tta2yta committed:
adding styles top menu

tta2yta committed:
adding styles top menu

tta2yta committed:
adding main-section and footer section

tta2yta committed:
adding styles to footer logo

tta2yta committed:
adding styles to footer edition section

tta2yta committed:
adding styles to footer edition section

tta2yta committed:
adding styles to footer edition section

tta2yta committed:
adding styles to footer edition section

tta2yta committed:
correcting linters

tta2yta committed:
apdating readme

tta2yta committed:
Merge branch 'feature' of https://github.com/tta2yta/thenextweek into feature

tta2yta committed:
adding styles to scrollable header

tta2yta committed:
Merge branch 'feature' of https://github.com/tta2yta/thenextweek into feature

tta2yta committed:
adding styles to top-menu-header

tta2yta committed:
adding styles to header-menu-2

tta2yta committed:
adding styles to header-menu-2

tta2yta committed:
adding styles to main section

tta2yta committed:
correcting linters

tta2yta committed:
adding main section column one content

tta2yta committed:
adding main section column one content

prolajumokeoni committed:
add footer links

tta2yta committed:
adding style to top stries section

tta2yta committed:
adding style to mt turn section

tta2yta committed:
adding style to culture and travel section

tta2yta committed:
Merge branch 'feature' of https://github.com/tta2yta/thenextweek into feature

tta2yta committed:
correcting linters

tta2yta committed:
Merge branch 'feature' of https://github.com/tta2yta/thenextweek into feature

SpaYco reviewed (changes requested):
# Invalid 🛑 

Hello, you've made great progress, but I can't review your work unless the requirements below are met 👇 

- [ ] The project should have a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))



Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

tta2yta committed:
Update README.md

prolajumokeoni committed:
Update README.md

add license

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

prolajumokeoni committed:
Update README.md

edit readme

tta2yta committed:
update readme

tta2yta committed:
Update README.md

tta2yta commented:
Hello. We have added the readme file. sorry, We forgot it.

mahmoud717 reviewed (changes requested):
### Changes required ♻️ 

**Hello @tta2yta  & @prolajumokeoni**
 

---

- You were to make a responsive ***Replica*** of the original website. that means that you have to have the **same** project layout, images, 
spacings, paddings, margins, and most importantly **content** with all of the sections present.
Please go back and do the required changes to make your replica look at least 95% like the original project.

- also make sure to fix your linters errors.

---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

prolajumokeoni commented:
> ### Changes required ♻️
> **Hello @tta2yta & @prolajumokeoni**
> 
> * You were to make a responsive _**Replica**_ of the original website. that means that you have to have the **same** project layout, images,
>   spacings, paddings, margins, and most importantly **content** with all of the sections present.
>   Please go back and do the required changes to make your replica look at least 95% like the original project.
> * also make sure to fix your linters errors.
> 
> please make these changes and request another code review.
> 
> if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

I submitted in error, I apologize for the mix up, this submission was supposed to be for the bootstrap project.

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+  background-color: #f72210;
+}
+
+.header-btn div {
+  float: right;
+  margin-bottom: 20px;
+}
```
- [ ] You should use the bootstrap class `float-right` here instead of css classes

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+.header-top-menu li a {
+  font-size: 17px;
+  color: black;
+  font-weight: bold;
```
- [ ] You should use the bootstrap class for font-weight and color here

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+    padding-bottom: 5px;
+  }
+
+  .fa-user {
+    display: inline;
+    margin-right: -50px;
+  }
+}
```
- [ ] You should use the bootstrap class `d-inline` here

BrittanyBlake replied:
- [ ] You need to use bootstrap classes here, you can use `d-*-inline`

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+    font-size: 12px;
+  }
+
+  .social-icons {
+    float: left !important;
+  }
```
- [ ] You can use the bootstrap class `float-left` here instead

BrittanyBlake replied:
- [ ] You need to use bootstrap classes here. You can use `float-*-left`

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+    height: 40px;
+  }
+
+  .login {
+    display: none;
+  }
```
- [ ] You should use the `d-none` bootstrap class here

BrittanyBlake replied:
- [ ] This still should be done using bootstrap classes. you can use `d-*-none` and replace the * with xs/sm/md/lg depending on what screen size you need it to display none in

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+}
+
+@media screen and (min-width: 992.99px) {
+  .header-main-2 {
+    display: none;
+  }
```
- [ ] You should use the `d-none` bootstrap class here

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+  position: relative;
+}
+
+.row.header-top {
+  padding: 0;
+  margin: 0;
+}
```
- [ ] You should use bootstrap classes `p-*` and `m-*` to set the margins and padding.

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+}
+
+header svg {
+  width: 100%;
```
- [ ] You can use `w-100` for this

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,293 @@
+  }
+
+  .header-btn div {
+    float: right;
```
- [ ] You can use the bootstrap class `float-right` here

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                            <div class="person-icon-acc">
+                                <a href="#" class="p-icon">
+
+                                    <i class="fa fa-fw fa-user" style="background-color: transparent; margin-right: -10px;"></i>
```
- [ ] The tablet screen size should say login, not the user icon. The user icon comes on the mobile screen size.

Your clone 👇🏾 
<img width="776" alt="Screen Shot 2021-01-25 at 10 38 29 PM" src="https://user-images.githubusercontent.com/48804258/105769397-0d526400-5f5e-11eb-8210-5b9540a2f271.png">
The original page 👇🏾 
<img width="778" alt="Screen Shot 2021-01-25 at 10 38 47 PM" src="https://user-images.githubusercontent.com/48804258/105769431-180cf900-5f5e-11eb-9210-c54296c31128.png">

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,260 @@
+                        © 2021 NEWSWEEK DIGITAL LLC
+                        </span>
+                  </div>
+                  <div class="col-md-4 col-lg-4 ">
+                      <div class="social-icons float-right">
+                        <a href="#" class="fa fa-facebook"></a>
+                        <a href="#" class="fa fa-twitter"></a>
+                        <a href="#" class="fa fa-instagram"></a>
+                        <a href="#" class="fa fa-tumblr"></a>
+                        <a href="#" class="fa fa-linkedin"></a>
+                      </div>
```
- [ ] Your icons don't look great right now. I understand if you cant find the same icons as used in the clone but maybe try resizing some of them so it looks better.

BrittanyBlake reviewed (changes requested):
### Status: Changes Required ♻️ 
****
### Hi @prolajumokeoni and @tta2yta! 👋 
You have done a wonderful job with your milestone so far! It looks really good! 👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ]  Your linter errors are failing. Please fix the errors reported on GitHub actions.
- [ ] You are using a lot of css styling that can be done with bootstrap classes. Please take a look at this [cheat sheet](https://hackerthemes.com/bootstrap-cheatsheet/) and see what rules you can replace with bootstrap classes. 
- See my comments below for additional changes

****
### Please note that milestone 1 does not require a re-review and will be marked as approved in your dashboard, however these changes are MANDATORY. Please make the required changes before moving on to Milestone 2 🙏
Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

prolajumokeoni committed:
add first section images and text

prolajumokeoni committed:
add image

tta2yta commented:
Hello. We have corrected the code reviews you mentioned and we have added three columns of the main content as of the milestone 2 schedules and have inserted some contents according to the original website. we kindly ask you to review our code until we have done yet. Thank you for your time.

tta2yta committed:
Update README.md

tta2yta committed:
Update README.md

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,315 @@
+
+footer svg,
+.header-top svg {
+  display: inline;
```
- [ ] You should use bootstrap classes for this.

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,315 @@
+
+.fa {
+  font-size: 14px;
+  text-decoration: none;
```
- [ ] You can use bootstrap classes for this. `text-decoration-none`.

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,315 @@
+
+.fa-user {
+  display: none;
+}
```
- [ ] You should use bootstrap classes for this.

BrittanyBlake commented on css/style.css:
```diff
@@ -0,0 +1,315 @@
+.feature-stories img,
+.top-stories img {
+  max-width: 100%;
+}
```
- [ ] You can use bootstrap classes for this: `mw-100`

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,588 @@
+                      </li>
+                     
+                    </ul>
+        </div>
```
- [ ] The bottom border is missing in this section on the **Desktop** screen size:

Your clone 👇🏾 
<img width="1236" alt="Screen Shot 2021-01-26 at 7 35 35 PM" src="https://user-images.githubusercontent.com/48804258/105888755-aab8a100-600d-11eb-84f4-78dfb918578e.png">

The original page 👇🏾 
<img width="1310" alt="Screen Shot 2021-01-26 at 7 36 03 PM" src="https://user-images.githubusercontent.com/48804258/105888797-bb691700-600d-11eb-9268-b9bc7d7d29ec.png">

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,588 @@
+        <section class="main-content bg-white border-bottom">
+            <div class="row mb-3 ">
+                <div class="feature-stories col-md-6 col-lg-3">
+                    <h5 class="pl-2 mt-3">FEATURE STORIES</h5>
```
- There is a typo in this section. It should be **Featured** but you have written **feature**. It is a small error but it would be nice to fix it.

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,588 @@
+                    <div class="card pl-2 mb-4 border-0">
+                        <div class="card-img-sec my-0 py-0">
+                            <img class="card-img-top my-0 py-0 " src="https://web.archive.org/web/20210120125504im_/https://d.newsweek.com/en/full/1711616/donald-trump.jpg?w=397&h=265&q=90&f=d759057143d283aa0ea0ed13fba5450a" alt="Card image cap">
+                        <h5 class="txt-in-image py-1 px-3 mb-0">US</h5>
```
- [ ] The text in the label in the image should be red. This can be done with the bootstrap class `text-danger`

Your clone 👇🏾 
<img width="387" alt="Screen Shot 2021-01-26 at 7 41 34 PM" src="https://user-images.githubusercontent.com/48804258/105889484-80b3ae80-600e-11eb-9075-e0d1890bff79.png">

The original page 👇🏾 
<img width="338" alt="Screen Shot 2021-01-26 at 7 41 09 PM" src="https://user-images.githubusercontent.com/48804258/105889435-72fe2900-600e-11eb-9e95-d6052fe1d936.png">

BrittanyBlake commented on index.html:
```diff
@@ -0,0 +1,588 @@
+
+                
+                <div class="top-stories col-12 col-md-6 col-lg-6">
+                    <h5 class="pl-2 mt-3 text-muted">TOP STORIES</h5>
```
- There is a typo in this section. It should be **Top Story** but you have written **Top Stories**. It is a small error but it would be nice to fix it.

### Mobile screen

- [ ] This section should come first on the mobile screen size. The order changes as the screen size changes.

Your clone 👇🏾 
<img width="429" alt="Screen Shot 2021-01-26 at 7 47 02 PM" src="https://user-images.githubusercontent.com/48804258/105890152-44348280-600f-11eb-83f7-78ea428a8b51.png">

The original page 👇🏾 
<img width="434" alt="Screen Shot 2021-01-26 at 7 47 31 PM" src="https://user-images.githubusercontent.com/48804258/105890200-56162580-600f-11eb-88af-aaf05670f36b.png">

Requirements from [Microverse](https://microverse.pathwright.com/library/new-technical-curriculum/177956/path/step/104328875/) 👇🏾 
<img width="861" alt="Screen Shot 2021-01-26 at 7 48 52 PM" src="https://user-images.githubusercontent.com/48804258/105890342-865dc400-600f-11eb-850e-f982b2f2fd44.png">

BrittanyBlake reviewed (changes requested):
### Status: Changes Required ♻️ 
****
### Hi @prolajumokeoni and @tta2yta! 👋 
Great job working on the requested changes!👏 👏 👏  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ]  Your linter errors are failing. Please fix the errors reported on GitHub actions.
- [ ] You are using a lot of css styling that can be done with bootstrap classes. Please take a look at this [cheat sheet](https://hackerthemes.com/bootstrap-cheatsheet/) and see what rules you can replace with bootstrap classes. 
- [ ] As was mentioned before, the tablet screen size should say login, it should not yet  have the user icon. The user icon comes on the mobile screen size.

Your clone 👇🏾 
<img width="779" alt="Screen Shot 2021-01-26 at 7 32 09 PM" src="https://user-images.githubusercontent.com/48804258/105888382-30881c80-600d-11eb-9592-398ededd4d22.png">

The original page 👇🏾 
<img width="797" alt="Screen Shot 2021-01-26 at 7 32 25 PM" src="https://user-images.githubusercontent.com/48804258/105888412-3978ee00-600d-11eb-9ef8-8c128eb1df78.png">

- [ ] The following sections are missing 
 **Column 2**:
    -  More stories
    - Latest news
   **Column 3**:
     - The debate
     - Opinion
     - Sponsored insights / Rankings
     - Free signup
- In future, try not to submit an incomplete project.

- See my comments below for additional changes


****
### Please note that milestone 2 does not require a re-review and will be marked as approved in your dashboard, however these changes are MANDATORY. Please make the required changes before moving on to Milestone 3 🙏
Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

tedtesande committed:
adding more stories section

tedtesande committed:
adding more stories section

tedtesande committed:
adding debate section

tta2yta committed:
addingfeatures to debate section

tta2yta committed:
adding opinion section

tta2yta committed:
adding opinion image and signup section

tta2yta committed:
adding styles to myurn and culture section

tta2yta committed:
adding styles sighnup section for sm

tta2yta committed:
adding styles to latest news

tta2yta committed:
adding styles to opinion section

tta2yta committed:
adding styles

tta2yta committed:
adding structure and styles to magazine

tta2yta committed:
adding editorila section
